Integrations/EclecticIQ/EclecticIQ.py

```diff
@@ -0,0 +1,640 @@
+''' IMPORTS '''
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import json
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS/PARAMS '''
+
+USERNAME = demisto.params().get('credentials').get('identifier')
+PASSWORD = demisto.params().get('credentials').get('password')
+URL = demisto.params()['url']
+SERVER = URL[:-1] if (URL and URL.endswith('/')) else URL
+USE_SSL = not demisto.params().get('insecure', False)
+HEADERS = {}
+IP_THRESHOLD = demisto.params().get('ip_threshold').lower()
+URL_THRESHOLD = demisto.params().get('url_threshold').lower()
+FILE_THRESHOLD = demisto.params().get('file_threshold').lower()
+EMAIL_THRESHOLD = demisto.params().get('email_threshold').lower()
+DOMAIN_THRESHOLD = demisto.params().get('domain_threshold').lower()
```

Comment:
loved how the name is ordered by length 😉

Reply:
worked hard on that one